includes/sys_menu.php

```diff
@@ -58,7 +58,7 @@ function make_navigation()
     $pages = [
         'news'           => __('news.title'),
         'meetings'       => [__('news.title.meetings'), 'user_meetings'],
-        'user_shifts'    => __('general.shifts'),
+        'user_shifts'    => __('general.my_shifts'),
```

Comment:
wait – but changing the menu was precisely my intention – at least for users (I see how the wording doesn't apply to admins 😅). Imo, this improves the discoverability of the menu.
keeping the menu item as "shifts", but then having the shift page heading be "my shifts" would imo just be an additional source of confusion because it's inconsistent.

I've implemented a suggestion on how to handle this in https://github.com/engelsystem/engelsystem/pull/1413/commits/bf0f3f374ec8fcf37d8c8544b931ca5129354202 , do you think this approach would be feasible? (I have 0 php knowledge, will be happy to research more elegant ways to implement this if you're happy with the general direction)

Reply:
Why do you want to change this page's title to "My shifts"? 

This page shows all (available) shifts. That's what I meant by "shifts overview" page. `/user-shifts`

<img width="1440" alt="Screenshot 2024-08-17 at 15 05 48" src="https://github.com/user-attachments/assets/0257199a-61b4-4f11-aa52-d71182b71b3b">


The page that shows all shifts you signed up for is your user profile `/users?action=view`

<img width="1438" alt="Screenshot 2024-08-17 at 15 05 08" src="https://github.com/user-attachments/assets/bd6bcf4e-b9fc-48d1-ab1c-2eefe49b23af">

I interpreted your comment "I've found it super confusing that "Shifts" does not show all available shifts, but rather the shifts I signed up for " to mean that you want to change the heading "Shifts" of the user page `/users?action=view` to "my shifts" for not admins, which is a good idea.

However changing the shifts overview page title to "My shifts" makes no sense, since this is an overview of all shifts (and avialability state) for all users.

Reply:
🤯 Oohh I think there's been both a miscommunication between us and a misunderstanding on our (organizations') part on how to see & sign up for shifts.
I've been solving the wrong UX bug all along! :D

So, from my current understanding, what's been happening is the following:
In our shift system, there are so many rooms that the list of them pushes the shift table off the screen. Additionally, Changing a date in the drop down menus does not change anything on the page (I just noticed now that you's need to click the "filter" button, which is different to the behaviour of the room page date drop-down menus). So, this overview looked empty to us:

<img width="1552" alt="Screenshot 2024-08-26 at 14 33 06" src="https://github.com/user-attachments/assets/afc64d93-190f-44e4-81fa-78ae2e04d278">


Which is why we instructed all of our helpers to use the "Rooms" drop down, pick an area of the festival they wanted to help with, and then sign up for a shift from there. This is not a path that's easy to discover on your own, so with this PR I wanted to guide users away from the path I *thought* was wrong – turns out it *was* right, the relevant information was just hiding off the screen 😅.

And this wasn't just me, it was *all* of our organization – I've been taught to go via the room view by peers, if I remember it correctly. (If you'd like to see for yourself, the shifts for august 9th-11th are still online at https://engel.wutzrock.de/)

my suggestion for next steps would be:
1. I'll re-work this PR to do what you interpreted it to be intended for ( change the heading "Shifts" of the user page /users?action=view to "my shifts" for not admins)
2. I'll open a separate PR implementing an immediate page update after the user changes something in the drop-down menu or clicks one of the "gestern","heute",.... buttons (just like the drop down menu in the room menu behaves), obsoleting the "Filter" button (at least for the time overview – if it's too slow to update on every room/angel type checkbox change, I'd visually & functionally move the Filter button there).
3. I start working on a proposal to re-arrange the Shifts page that the shifts aren't pushed so far down that you can't see them 

What do you think?

Reply:
The suggestions sound good, but we have to make sure that its not making the problem "worse" (for example side effects when users add themselves to an angel type and don't see the filter to add them there too)

Do you want to create a new issue for this to discuss a possible solution?